src/vote_stage.rs

```diff
@@ -74,9 +74,10 @@ pub fn send_validator_vote(
 ) -> Result<()> {
     let last_id = bank.last_id();
 
-    let shared_blob = create_new_signed_vote_blob(&last_id, vote_account, bank, cluster_info)?;
-    inc_new_counter_info!("validator-vote_sent", 1);
-    vote_blob_sender.send(vec![shared_blob])?;
-
+    if let Ok(shared_blob) = create_new_signed_vote_blob(&last_id, vote_account, bank, cluster_info)
```

Comment:
Filed an issue to store failed votes: https://github.com/solana-labs/solana/issues/2152